core/src/main/java/com/novoda/noplayer/exoplayer/forwarder/EventInfoForwarder.java

```diff
@@ -0,0 +1,78 @@
+package com.novoda.noplayer.exoplayer.forwarder;
+
+import com.google.android.exoplayer2.ExoPlaybackException;
+import com.google.android.exoplayer2.ExoPlayer;
+import com.google.android.exoplayer2.PlaybackParameters;
+import com.google.android.exoplayer2.Timeline;
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.TrackSelectionArray;
+import com.novoda.noplayer.listeners.InfoListeners;
+
+import java.util.HashMap;
+
+class EventInfoForwarder implements ExoPlayer.EventListener {
```

Comment:
At some point this will likely contain more implementations, being an `info` forwarder, it is the most generic reporter we have, mainly useful for debugging. 🤷‍♂️